src/main/java/ziwg/czy_dojade_backend/services/implementations/DataService.java

```diff
@@ -0,0 +1,293 @@
+package ziwg.czy_dojade_backend.services.implementations;
```

Comment:
może nazwać to jakoś DataImportService czy coś w tym stylu? DataService to za bardzo nie mówi nic